src/coreclr/jit/simd.h

```diff
@@ -415,18 +430,7 @@ inline double EvaluateUnaryScalarSpecialized<double>(genTreeOps oper, double arg
 template <typename TBase>
 TBase EvaluateUnaryScalar(genTreeOps oper, TBase arg0)
```

Comment:
nit: is `EvaluateUnaryScalar` method needed then?

Reply:
It might not be needed, but it's consistent with the setup of other APIs.

If we were to change this, I'd probably make it `EvaluateUnaryScalar` and just specialize that directly instead. That way there isn't two differently named entry points for Unary vs Binary vs ...